ddtrace/contrib/aiobotocore/__init__.py

```diff
@@ -1,10 +1,7 @@
 """
-The aiobotocore integration will trace all AWS calls made with the `aiobotocore`
-library.
-
-This integration ignores autopatching, it can be enabled via
-`patch_all(botocore=True)`
-::
+The aiobotocore integration will trace all AWS calls made with the ``aiobotocore``
+library. This integration ignores autopatching, and can be enabled via
```

Comment:
This is confusing wording "This integration ignores autopatching".
We should instead say:

> This integration isn't enabled when applying the default patching. To enable it, you must run `patch_all(botocore=True)`.